src/App.js

```diff
@@ -4,17 +4,23 @@ import timelineData from './data/timeline.json';
 
 import Timeline from './components/Timeline';
 
+import TimelineEvent from './components/TimelineEvent';
+import Timestamp from './components/Timestamp';
+
 class App extends Component {
   render() {
-    console.log(timelineData);
 
     // Customize the code below
     return (
       <div className="App">
         <header className="App-header">
-          <h1 className="App-title">Application title</h1>
+          <h1 className="App-title">Ada Lovelace's Social Media Feed</h1>
         </header>
         <main className="App-main">
+          <ul>
+
+            <Timeline eventProps={timelineData.events}/>
```

Comment:
I love how you pass through the data here so you can manipulate it in more nested components.